CHANGES.md

```diff
@@ -7,6 +7,8 @@ Version 0.1.1
 To be released.
 
  -  Relicensed under [LGPL 2.1] or later.
+ -  Added more implicit operators of `Integer`, `Boolean`, `Binary`,
+    `Dictionary`.
```

Comment:
Could you write list of target types that these implicit conversions (plus: we'd better name them “implicit conversions” than “implicit operators”) return?  It could be like “`Integer` ↔︎ `short`, `Integer` ↔︎ `int`, …”  As it lengthens it's better if it's a bullet list.